packages/browser/src/errors.ts

```diff
@@ -0,0 +1,33 @@
+import { NormalizeKeyPaths } from '@silverhand/essentials';
+import get from 'lodash.get';
+
+const logtoClientErrorCodes = Object.freeze({
+  sign_in_session: {
+    not_found: 'Sign-in session not found.',
+  },
+});
+
+export type LogtoClientErrorCode = NormalizeKeyPaths<typeof logtoClientErrorCodes>;
+
+const getMessageByErrorCode = (errorCode: LogtoClientErrorCode): string => {
+  // TODO: linear issue LOG-1419
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const message = get(logtoClientErrorCodes, errorCode);
```

Comment:
do we really need `lodash.get` here? seems this can work:
```ts
const message = logtoClientErrorCodes[errorCode];
```

Reply:
actual struct like this, cannot get the error message simply by one key.

<img width="833" alt="image" src="https://user-images.githubusercontent.com/10594507/153986377-626a63e4-e599-42bc-9dd7-5b1452ffced3.png">